v2/yarpcgrpc/outbound.go

```diff
@@ -80,7 +80,7 @@ func (o *Outbound) Call(ctx context.Context, req *yarpc.Request, reqBuf *yarpc.B
 
 	return &yarpc.Response{
 			Headers:          responseHeaders,
-			ApplicationError: metadataToIsApplicationError(responseMD),
+			ApplicationError: metadataToApplicationError(invokeErr, responseMD),
```

Comment:
This is where I hope we eventually take advantage of a DecodeError method from the encoding to extract error details from the transport metadata. Since we’ve not yet done outbound encoding middleware, it’s not clear how that would be carried to this point.